training/model_management/src/run_model_preprocess.py

```diff
@@ -59,5 +61,12 @@ def _validate_hf_transformers_args(args):
         _validate_hf_transformers_args(preprocess_args)
 
     run_preprocess(mlflow_flavor, model_path, mlflow_model_output_dir, **preprocess_args)
+
+    ## Adding support for license file if exist in model path
+    if os.path.exists(Path(model_path,LICENSE_FILE)):
+        shutil.copy(Path(model_path,LICENSE_FILE), mlflow_model_output_dir)
```

Comment:
This is specific to hugging face conversion. Please add changes in  hftransformers/convert.py

Reply:
Done